### DISCUSSION TITLE
Tailwind doesn't override the default colors.

### DISCUSSION THREAD

**Discussion by april-ctrlspire** [tailwindlabs/tailwindcss#13572] [Help]
I'm using the latest Tailwind version. In the docs, it says "If you’d like to completely replace the default color palette with your own custom colors, add your colors directly under the theme.colors".  I have tried that and it doesn't work.

Here's my custom colors:

```javascript
theme: {
    colors: {
      transparent: 'transparent',
      current: 'currentColor',
      white: '#ffffff',
      gray: {
        DEFAULT: '#0A1928',
        700: '#545E69',
      },
      black: {
        DEFAULT: '#0A1928',
        400: '#999999',
        800: '#333333',
      },
      blue: {
        DEFAULT: '#0F62FE',
        800: '#3F81FE',
      },
    },
    extend: {},
  },
``` 

If I use **text-blue**, it shows my custom color: #0F62FE, but if i try **text-blue-800**, it shows the default color: #1e40af, NOT my custom color: #3F81FE.

Why does the variants like 800 does not work? Is there another way to completely replace the default colors?

**Comment by andymccray** [comment]
Seems to work okay in a sandbox example: https://play.tailwindcss.com/O1K08YHVAW

Can you share the entire config file?

**Reply by april-ctrlspire** [reply]
Thanks for the reply.

Here's the whole config file. I'm using wedges UI and I removed it and it still won't work.

```javascript
import type { Config } from 'tailwindcss';
import { wedgesTW } from '@lemonsqueezy/wedges';

export default {
  content: [
    './app/**/*.{js,jsx,ts,tsx}',
    'node_modules/@lemonsqueezy/wedges/dist/**/*.{js,ts,jsx,tsx}',
  ],
  theme: {
    colors: {
      transparent: 'transparent',
      current: 'currentColor',
      white: '#ffffff',
      gray: {
        DEFAULT: '#0A1928',
        700: '#545E69',
        900: '#121217',
      },
      black: {
        DEFAULT: '#0A1928',
        400: '#999999',
        800: '#333333',
      },
      blue: {
        DEFAULT: '#0F62FE',
        800: '#3F81FE',
      },
      neutral: {
        DEFAULT: '#F4F6F9',
        50: '#D9D9D9',
        100: '#EDF0F5',
        500: '#CDCFD4',
        600: '#D1D1DB',
      },
    },
    extend: {},
  },
  darkMode: 'class',
  plugins: [require('@tailwindcss/typography'), wedgesTW()],
} satisfies Config;

```

**Reply by april-ctrlspire** [reply]
I'm using Remix and made a fresh install and it works fine. I think the Remix I used has this issue with Tailwind. I'm closing this.
